data/mods/Magiclysm/items/ethereal_items.json

```diff
@@ -282,12 +282,37 @@
     "volume": 10,
     "bashing": 16,
     "to_hit": 1,
-    "armor_data": { "covers": [ "TORSO" ], "coverage": 5, "material_thickness": 1, "encumbrance": 10 },
```

Comment:
So no more wearing Woodbow now?

Reply:
Well having this on makes it summon to being worn, and I don't really think it's around long enough to justify wearing it. Either you wanna shoot stuff with it or you don't, but I'm flexible with it staying if that's something people use. I don't exactly get any feedback on how people use these spells.

Reply:
i had noticed that when i was fixing the firing error the other day. i feel the same way: i think it probably isn't around long enough for you to really worry about it, and you can just drop it on the ground because you can summon a new one if it's that important to save space.

Reply:
Speaking of, it may need to be adjusted for how long it's around, I just intended it to be a short term combat spell, enough to let you engage in a short fight for a few minutes. That concept of a 'few minutes' in moves has changed now though.